tck/scala-tck/src/main/scala/com/akkaserverless/scalasdk/Access.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.akkaserverless.scalasdk
+
+import akkaserverless.tck.model.{ Components, ComponentsImpl }
+import com.akkaserverless.javasdk.impl
+import com.akkaserverless.scalasdk.impl.InternalContext
+import com.akkaserverless.scalasdk.impl.valueentity.ScalaValueEntityContextAdapter
+import com.akkaserverless.scalasdk.valueentity.ValueEntityContext
+
+object Access {
+  // FIXME: Access to components has been restricted in Scala SDK, see https://github.com/lightbend/akkaserverless-java-sdk/issues/673
+  // On the Scala side there's no way to make calls right now. The TCK, however, currently requires those calls, so we need this hack.
+  // Ultimately, these tests are probably removed from the SDK at some point, in which case we can remove this method again.
+  def components(context: ValueEntityContext): Components =
```

Comment:
Quick hack to allow to access other components from more places.

Reply:
Hmm, the intent was that they should now be available in all entities as well as actions but maybe  I missed opening that up in the codegen for Scala and only did it in Java?

The ticket was about discussing maybe limiting that part in the future rather (but maybe not).

Reply:
#673 sounds as if you had only added `components` support for the Java side in #628  because the TCK required it.

Reply:
But anyway if you say the discrepancy is an oversight, let's create a ticket to fix that?

Reply:
https://github.com/lightbend/akkaserverless-java-sdk/pull/689

Reply:
Or a PR :)

Reply:
Even better :) Will update after that one is merged.